requirements.txt

```diff
@@ -1,6 +1 @@
 colored
-mypy>=0.981
-mypy-extensions>=0.4.3
-tomli>=2.0.1
-types-setuptools>=65.4.0.0
-typing_extensions>=4.3.0
```

Comment:
Again just a proposition. It seems enough just to have `colored` installed to run the app, the rest is only for developers maintaining the code. `Poetry` separates dev requirements nicely but I didn't have a better idea to do it with `pip`.

Reply:
You already did everything right with using `requirements-dev.txt` for that.